Makefile.pseudomodules

```diff
@@ -23,3 +23,6 @@ PSEUDOMODULES += netif
 # include variants of the AT86RF2xx drivers as pseudo modules
 PSEUDOMODULES += at86rf23%
 PSEUDOMODULES += at86rf21%
+
+# add all pseudo random number generator variants as pseudomodules
+PSEUDOMODULES += prng_%
```

Comment:
(I would prefer keeping it or renaming to `prand` or somethieng else, since PRNG isn't a abbreviation lot of people (even in computer science) know necessarily, as far as I experienced it)